programs/util/pem2der.c

```diff
@@ -185,6 +185,14 @@ int main( int argc, char *argv[] )
     size_t pem_size, der_size = sizeof(der_buffer);
     int i;
     char *p, *q;
+#if defined(MBEDTLS_PLATFORM_C)
+    mbedtls_platform_context platform_ctx;
+    if( ( ret = mbedtls_platform_setup( &platform_ctx ) ) != 0 )
+    {
+        mbedtls_printf( " failed\n  !  mbedtls_platform_setup returned %d\n\n", ret );
+        goto exit;
+    }
```

Comment:
do we need platform initialization for this example? We don't have an alternative implementation for PEM \ DER parsing, and I don't see it happenning in the near future

Reply:
I am not sure about this approach. I can imagine someone wanting to run any on a specific platform. In such case all the examples should be prepared for that. Do you think otherwise?

Reply:
Even if this example runs on the platform, no cryptographic  platform initialization needs to be done for PEM \ DER, as it is SW only code.